src/Webform.js

```diff
@@ -1189,7 +1189,7 @@ export default class Webform extends NestedDataComponent {
       errors = super.errors;
     }
 
-    errors = errors.concat(this.customErrors);
+    errors = errors.concat(this.serverErrors);
```

Comment:
I didn't find the case when it breaks but I'm aware you might be right so I suppose I have to leave both custom and server Errors here in case of preventing any unexpected behavior. 

> Please confirm that all tests passed locally.

All tests passed locally.